pkg/network/java/testdata/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
```

Comment:
you should add the path of docker-compose to `save_test_dockers` command in `system-probe.py` in order to pull the image in the CI, and especially the images that you want to pull..